tests/v2/test_auth.py

```diff
@@ -13,6 +13,54 @@ def test_create_user(self):
         response_content =  json.loads(response.data.decode())
         self.assertTrue(response_content['status'] == 201)
 
+    def test_user_login(self):
+        """Test that endpoint can login a registered user"""
+        super().create_user(admin_user)
+        response = super().login_user(admin_user_login)
+        self.assertEqual(response.status_code, 200)
+        response_content =  json.loads(response.data.decode())
+        self.assertTrue(response_content['status'] == 200)
+
+    def test_wrong_password_login(self):
+        """Test that endpoint can login a registered user"""
+        super().create_user(admin_user)
+        response = super().login_user(wrong_password_login)
+        self.assertEqual(response.status_code, 401)
+        response_content =  json.loads(response.data.decode())
+        self.assertTrue(response_content['status'] == 401)
+
+    def test_blank_password_login(self):
+        """Test that endpoint can login a registered user"""
+        super().create_user(admin_user)
+        response = super().login_user(blank_password_login)
+        self.assertEqual(response.status_code, 400)
+        response_content =  json.loads(response.data.decode())
+        self.assertTrue(response_content['status'] == 400)
+
+    def test_blank_password_login(self):
+        """Test that endpoint can login a registered user"""
+        super().create_user(admin_user)
+        response = super().login_user(blank_password_login)
+        self.assertEqual(response.status_code, 400)
+        response_content =  json.loads(response.data.decode())
+        self.assertTrue(response_content['status'] == 400)
+
+    def test_unexisting_login(self):
+        """Test that endpoint can login a registered user"""
+        super().create_user(admin_user)
+        response = super().login_user(blank_password_login)
+        self.assertEqual(response.status_code, 403)
+        response_content =  json.loads(response.data.decode())
+        self.assertTrue(response_content['status'] == 401)
+
+    def test_blank_body_login(self):
+        """Test that endpoint can login a registered user"""
```

Comment:
The `doc strings` in all of your tests are not describing what is happen in the methods, they are talking about an endpoint can register a user and yet we testing the login endpoint. Kindly fix that.